lib/vdsm/virt/vm.py

```diff
@@ -89,14 +88,14 @@
 from vdsm.virt import blockjob
 from vdsm.virt import cpumanagement
 from vdsm.virt import domxml_preprocess
-from vdsm.virt import drivemonitor
 from vdsm.virt import errors
 from vdsm.virt import guestagent
 from vdsm.virt import libvirtxml
 from vdsm.virt import metadata
 from vdsm.virt import migration
 from vdsm.virt import sampling
 from vdsm.virt import saslpasswd2
+from vdsm.virt import thinp
```

Comment:
I'm not sure `thinp' is the best / most clear name, but perhaps it's a storage jargon?

Reply:
Its a shortcut of thin provisioning, which is what be basically do here.
Other names I considered are volmonitor.py and chunked.py, but I don't like
names that repeat the class name, and "chunked" is not a common storage term.